R/utilities-scenarios.R

```diff
@@ -16,31 +15,46 @@
 #' `SimulatioResults` objects produced by running the simulation, and output values
 #' of the `SimulationResults`.
 #' @export
-runScenarios <- function(scenarioNames, scenarioConfiguration, customParams = NULL,
+runScenarios <- function(scenarioConfigurations, customParams = NULL,
```

Comment:
Here, we have to pass scenario configurations ready to run instead of initializing them within the functin.